react-common/styles/share/share.less

```diff
@@ -314,7 +315,46 @@
     }
 }
 
-@media @mobileAndBelow {
+@media only screen and (max-width: 854px) {
```

Comment:
these numbers seem really arbitrary. what are they based on?

Reply:
The 854 was an observed breakpoint, but on further testing, it looks like the more accurate measurement was the 991 breakpoint that we have for tablets.

The 513 is just for 512px screen widths; this is particularly for https://github.com/microsoft/pxt-microbit/issues/5468. When a screen reaches 512px in width on a 1280x720px resolution screen, making the modal break into the mobile view is not ideal and hard to work with. There needs to be an in-between before the content on the share modal orients as a column and the styling here is meant to help bridge that. This might be something that can also be removed in favor of a re-design, though.